src/components/GettingStartedDropdown/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { ChevronDownIcon } from "@chakra-ui/icons";
+import {
+  Button,
+  Menu,
+  MenuButton,
+  MenuDivider,
+  MenuGroup,
+  MenuItem,
+  MenuList,
+} from "@chakra-ui/react";
+import { GETTING_STARTED } from "../../constants/links";
+import { ExternalLink } from "../ExternalLink";
+
+export default function GettingStartedDropdown(): JSX.Element {
+  return (
+    <Menu colorScheme="#002954">
+      <MenuButton
+        as={Button}
```

Comment:
The button should be probably be using the `link` variant - https://chakra-ui.com/docs/form/button#button-variants

Reply:
Why is this a link variant? The figma shows a button as the component

Reply:
Are we looking at the same one? 😅 
![Screen Shot 2021-07-01 at 10 28 08 AM](https://user-images.githubusercontent.com/8749859/124150338-1688b500-da57-11eb-9bf0-32e3a1d90826.png)

Reply:
![Uploading Screen Shot 2021-07-01 at 9.27.25 AM.png…]()